### ISSUE TITLE
[LOC] Invalid jump link in an article

### ISSUE THREAD

**Issue by OfficeGSX** [MicrosoftDocs/OfficeDocs-SharePoint#192]
Hello there is an jump link which does not work.

**Repro steps**:
1. Go to https://docs.microsoft.com/en-us/sharepoint/administration/restrict-or-enable-access-to-a-service-application
2. Search for "To start a Windows PowerShell session"
3. Click on the jump link

**Actual results**:
Link does not jump
**Expected results**:
Link behaves as expected

**GH Article**:
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-pr/blob/live/SharePoint/SharePointServer/administration/restrict-or-enable-access-to-a-service-application.md

![image](https://user-images.githubusercontent.com/16706698/44148306-19418624-a098-11e8-98d3-0877b9062100.png)

**Comment by AndreaBarr** [comment]
@OfficeGSX Thank you for this feedback

I want to make sure this link is corrected. I am makeing sure that the SharePoint team gets this so they can address it properly.

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
